### PR TITLE
[CRM457-1143] Fix back button functionality from index

### PR DESCRIPTION
**Pull request by dwhenry**
issues related to loading tabs with turboframes resulting in bad caching

## Description of change

 [Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1143)

### CODE REVIEW

**Review by jsugarman — APPROVED** [review]
LGTM

File: app/views/prior_authority/applications/index.html.erb [hunk 1]
```diff
@@ -24,20 +24,20 @@
...
         <% c.with_tab(label: t(".tabs.submitted")) do %>
-          <%= turbo_frame_tag 'submitted_applications', src: submitted_prior_authority_applications_path do %>
+          <%= turbo_frame_tag 'submitted_applications', src: submitted_prior_authority_applications_path, loading: :lazy do %>
             <%= render 'prior_authority/applications/frame_contents/submitted', model: @submitted_model, pagy: @submitted_pagy %>
           <% end %>
         <% end %>
         <% c.with_tab(label: t(".tabs.drafts")) do %>
-          <%= turbo_frame_tag 'draft_applications', src: draft_prior_authority_applications_path do %>
-            <%= render 'prior_authority/applications/frame_contents/draft', model: @draft_model, pagy: @draft_pagy %>
+          <%= turbo_frame_tag 'draft_applications', src: draft_prior_authority_applications_path, loading: :lazy %>
```
Comment (patrick-laa) [inline app/views/prior_authority/applications/index.html.erb:39]: Is this working for you? I'm running it locally and getting this:
Screenshot 2024-02-26 at 17 20 02

Comment (patrick-laa) [reply app/views/prior_authority/applications/index.html.erb:39]: Looks to me that you have a `do` in the wrong place.

Comment (dwhenry) [reply app/views/prior_authority/applications/index.html.erb:39]: That is on me, I made a change at the end to make it work with JS disabled and didn't double check the view...

I had put the `do` on the wrong line which is why it rendered... this is now fixed.

**Review by patrick-laa — APPROVED** [review]

**Pull request merged**